src/commands/local-store/local-store-command.js

```diff
@@ -1,23 +1,28 @@
+import { setTimeout } from 'timers/promises';
 import {
     OPERATION_ID_STATUS,
     ERROR_TYPE,
     LOCAL_STORE_TYPES,
     PENDING_STORAGE_REPOSITORIES,
     TRIPLE_STORE_REPOSITORIES,
+    LOCAL_INSERT_MAX_ATTEMPTS,
```

Comment:
I would just make the naming for these variables more clear since it's only used for curated paranets

Reply:
Done